pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -239,12 +245,127 @@ func (adc *attachDetachController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	err := adc.populateActualStateOfWorld()
+	if err != nil {
+		glog.Errorf("Error populating the actual state of world: %v", err)
+	}
+	err = adc.populateDesiredStateOfWorld()
+	if err != nil {
+		glog.Errorf("Error populating the desired state of world: %v", err)
+	}
 	go adc.reconciler.Run(stopCh)
 	go adc.desiredStateOfWorldPopulator.Run(stopCh)
 
 	<-stopCh
 }
 
+func (adc *attachDetachController) populateActualStateOfWorld() error {
+	glog.V(5).Infof("Populating ActualStateOfworld")
+	nodes, err := adc.nodeLister.List(labels.Everything())
+	if err != nil {
+		return err
+	}
+
+	for _, node := range nodes {
+		nodeName := types.NodeName(node.Name)
+		for _, attachedVolume := range node.Status.VolumesAttached {
+			uniqueName := attachedVolume.Name
+			err = adc.actualStateOfWorld.MarkVolumeAsAttached(uniqueName, nil, nodeName, attachedVolume.DevicePath)
```

Comment:
The `nil` value for `volumeSpec` scares me like @gnufied mentioned earlier. But it sounds like (https://github.com/kubernetes/kubernetes/pull/39732#discussion_r107663462) you are confident it won't break anything. 

nit: when passing in a value and it is unclear what the parameter is add the parameter name as a inline comment, e.g.:
``` GO
err = adc.actualStateOfWorld.MarkVolumeAsAttached(uniqueName, nil /* volumeSpec */, nodeName, attachedVolume.DevicePath)

```

nit: add a comment why you think it will be ok to pass in nil (e.g. https://github.com/kubernetes/kubernetes/pull/39732#discussion_r107663462).